src/api/app/models/event/base.rb

```diff
@@ -169,6 +169,13 @@ def perform_create_jobs
       save if self.undone_jobs.positive?
     end
 
+    def mark_job_done!
```

Comment:
This method is really similar to this code in `CreateJob`:

https://github.com/openSUSE/open-build-service/blob/1379e7286ef419c56bc4f6f71a95bbdda79d2400/src/api/app/jobs/create_job.rb#L10-L14

I think this could be aligned.

Reply:
[Done](https://github.com/openSUSE/open-build-service/pull/11124/files#diff-f2926e4c685d321e3d5c4d684167eab3511f970c7aa0b3aced8406eb6123de66R12)